src/quo/components/utilities/token/loader.cljs

```diff
@@ -3,12 +3,13 @@
   (:require [clojure.string :as string]))
 
 (def ^:private tokens (loader/resolve-tokens))
+(def ^:private safe-lower-case (comp string/lower-case str))
```

Comment:
@ulisesmac, I think ideally this function should've been defined in `utils.string`. There we already have other `safe-*` functions.